Engines/Wine/QuickScript/Steam Script/script.js

```diff
@@ -130,6 +130,8 @@ module.default = class SteamScript extends QuickScript {
             .wizard(setupWizard)
             .prefix(this._name, this._wineDistribution, this._wineArchitecture, this._wineVersion);
 
+        this._environment = this._environmentFunc(wine);
```

Comment:
Why was this line added?

Reply:
Set the `_environment `variable in .go() from the `_environmentFunc`, since it cannot be set in `.environment()` as it could need the wine object. 

This PR still needs to add this behaviour for all `QuickScript`.

Reply:
I would prefer if we don't set the variable in object scope but in method scope, i.e.: 

```javascript
const environment = this._environmentFunc(wine);
```

If we do this we will need to forward `environment` to all methods that use it, i.e. the shortcut creation method

Reply:
It is easier in terms of code change that we set it as object change (like it was done before using .environment() method). I guess it is a matter of personal opinion ^^.

Reply:
If you set everything in object scope you kind of polute the object. In my opinion an action should be repeatable. This means that if you call `.go()` twice on the same installer it should lead to the same result. But if you write intermediate results in object scope and maybe even overwrite existing values then this will not be the case anymore.

Reply:
You do not overwrite an existing value since _environment was not defined. If you run agian .go() you will overwrite that value, but since it can depend on the wine object it could potentially change, so this is not problem.

Anyway I do not really care how it is done at this point, that is why I prefer the solution that needs the less code modifying/refactor. But you can do how you want.